src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java

```diff
@@ -74,7 +74,7 @@ public void hasMatcherDescriptionForFailedTest() {
         this.exception.expectMessage(
             String.format(
                 // @checkstyle LineLength (1 line)
-                "Expected: Scalar with \"something\"%n but was: \"something else\""
+                "Expected: Scalar with \"something\"%n but was: was \"something else\""
```

Comment:
@andreoss why do we have this duplicate was and what should be done to avoid it?

Reply:
@victornoel This is how Matchers compose in terms of descriptions. 
A second "was" is coming from `IsEqual` (i.e `BaseMatcher`) now.
I would remove "was" from `Assertion` and rely on `describeMismatch` for providing this part of a sentence (as `Matchers.assertThat` does). 
Can it be left as a todo in order to avoid futher snowballing of this PR?

Reply:
@andreoss that is very good idea, please do that

Reply:
@victornoel Done